pages/_app.tsx

```diff
@@ -75,8 +87,32 @@ const MyApp = ({ Component, pageProps }: AppProps) => {
         pathWithoutLocale={pathWithoutLocale}
       >
         <ThemeProvider disableSmoothScrolling={disableSmoothScrolling} headComponent={Head}>
-          <Layout>
-            <Component {...pageProps} />
+          <div
+            sx={{
+              position: 'absolute',
+              left: 0,
+              right: 0,
+              top: 0,
+              minHeight: '768px',
+              backgroundImage: `url('${process.env.BASE_PATH}/img/page-background.png')`,
+              backgroundSize: 'cover',
+              backgroundPosition: 'center top',
+              '@media (min-width: 1440px)': {
+                aspectRatio: '1440/768',
+              },
+            }}
+          />
+          <Layout
+            headerSticky
+            headerContent={
+              <NavigationMarketing activeRoute="/docs" NextLink={NextLink} rightAlignItems={[localeSwitcher]} />
+            }
+            mainContainer
+            footerContent={<Footer localeSwitcher={localeSwitcher} />}
+          >
+            <div>
```

Comment:
Hey @alexandriaroberts do you remember why you wrapped `<Component {...pageProps} />` in a `<div>` here? It doesn't seem required, but maybe I'm missing something.

Reply:
To be honest, I don't remember now but I think it was left there by mistake. It isn't necessary there. Thank you very much for pointing it out. Removing it. 🙂